websites/T/Tiermaker/dist/metadata.json

```
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "tiermaker",

```

```suggestion
    "service": "Tiermaker",
```
The service name must match the folder name. And it seems like the service and folder name should be "TierMaker".

---

So this is what you meant to suggest probably
```suggestion
    "service": "TierMaker",
```